mobile/lib/providers/app_life_cycle.provider.dart

```diff
@@ -35,7 +35,7 @@ class AppLifeCycleNotifier extends StateNotifier<AppLifeCycleEnum> {
     return state;
   }
 
-  void handleAppResume() {
+  void handleAppResume() async {
```

Comment:
I think most of the awaits in this function can be done in parallel, which will let the app resume more quickly.

Reply:
Yeah, we probably can do that in a separate PR